src/logic/install.ts

```diff
@@ -609,16 +609,19 @@ export async function installProfiles(
     options?: InstallOptions;
   },
   { logger, userError }: { logger: ILogger; userError: UserError }
-): Promise<void> {
+): Promise<{ continueWithInstall: boolean }> {
   const loadedResult = await SuperJson.load(joinPath(superPath, META_FILE));
   const superJson = loadedResult.match(
     v => v,
     err => {
       logger.error('errorMessage', err.formatLong());
 
-      return new SuperJson({});
+      return undefined;
     }
   );
+  if (!superJson) {
```

Comment:
nitpick:
```suggestion
  if (superJson === undefined) {
```

Reply:
or maybe rather don't use the match, but just `if (superJson.isErr())` :thinking: